client/packages/system/src/IndicatorsDemographics/DetailView/GrowthRow.tsx

```diff
@@ -95,18 +98,27 @@ export const GrowthRow = <T extends RecordWithId>({
                       borderBottom: 'none',
                       alignItems: 'center',
                       display: 'flex',
+                      justifyContent: 'flex-end',
                     }}
                   >
-                    {hasColumnText ? <>% Growth on previous year</> : null}
+                    {hasColumnText ? t('label.growth-on-previous-year') : null}
                     {columnHeader ? (
-                      <BasicTextInput
-                        defaultValue={columnHeader.value ?? 0}
-                        onBlur={e =>
-                          setData({
-                            id: columnHeader.id,
-                            value: Number(e.target.value),
-                          })
-                        }
+                      <NumericTextInput
+                        value={columnHeader.value ?? 0}
+                        decimalLimit={2}
+                        decimalMin={1}
+                        InputProps={{
+                          endAdornment: (
+                            <InputAdornment position="end">%</InputAdornment>
+                          ),
+                        }}
```

Comment:
```suggestion
                        InputProps={{
                          inputProps: { sx: { padding: '2px 0' } },
                          endAdornment: (
                            <InputAdornment position="end">%</InputAdornment>
                          ),
                        }}
```

nit: adding padding something like this will make the input value and the `%` sign line up vertically 😅